### PR TITLE
fixed streaming for new models

### PR DESCRIPTION
**Pull request by maisiukartyom**

### CODE REVIEW

File: flow_prompt/ai_models/gemini/gemini_model.py [hunk 1]
```diff
@@ -102,11 +102,10 @@ def call(self, messages: t.List[dict], max_tokens: int, **kwargs) -> AIResponse:
         content = ""
         
         try:
-            if kwargs.get('stream'):
+            if not kwargs.get('stream'):
```
Comment (Katsiarynka) [inline flow_prompt/ai_models/gemini/gemini_model.py:105]: why is it different?

Comment (maisiukartyom) [reply flow_prompt/ai_models/gemini/gemini_model.py:105]: I mixed up the if statements. It was streaming when shouldn't and vice versa

File: tests/prompts/test_stream.py [hunk 2]
```diff
@@ -28,14 +28,15 @@ def gpt4_behaviour():
...
                     max_tokens=C_128K,
                     support_functions=True,
-                    should_verify_client_has_creds=False,
+                    should_verify_client_has_creds=False
                 ),
                 weight=100,
-            ),
+            )
         ]
     )
```
Comment (Katsiarynka) [inline tests/prompts/test_stream.py:42]: can you add tests? Please add in template of PR a question if a person added a mechanism, if it was bug-fix

**Review by Katsiarynka — APPROVED** [review]

File: flow_prompt/prompt/flow_prompt.py [hunk 3]
```diff
@@ -3,25 +3,26 @@
...
 from flow_prompt.prompt.pipe_prompt import PipePrompt
 from flow_prompt.prompt.user_prompt import UserPrompt
 from flow_prompt.responses import AIResponse
 from flow_prompt.services.flow_prompt import FlowPromptService
 from flow_prompt.utils import current_timestamp_ms
+import json
 
 logger = logging.getLogger(__name__)
 
+FLOW_PROMPT_URL = 'https://api.qa.flow-prompt.com'
```
Comment (Katsiarynka) [inline flow_prompt/prompt/flow_prompt.py:24]: should be taken from env if it's custom. Please delete, and add in the readme;

**Review by Katsiarynka — DISMISSED** [review]
Please make sure to delete:
```
FLOW_PROMPT_URL = 'https://api.qa.flow-prompt.com'
```

**Pull request merged**